vm-image-spec.yaml

```diff
@@ -244,6 +245,20 @@ files:
         query: |
           select neon.approximate_working_set_size(false) as approximate_working_set_size;
 
+      - metric_name: lfc_approximate_working_set_size_seconds
```

Comment:
`_seconds` suffix usually hints at the type of metric, which are `bytes`. I think smth like `lfc_approximate_working_set_size_per_window` could work better.

But it is not a huge problem, if we already depend on the new name, I think it can be left as-is.

Reply:
I think there's nothing merged that depends on the new name; we can definitely change it :smile:

Maybe `lfc_approximate_working_set_size_windows`? (kind of like `_buckets` for histograms)

Reply:
ref
- https://prometheus.io/docs/practices/naming/
- https://prometheus.io/docs/practices/histograms/

Based on the prometheus docs, I think the most fitting name here is something like `lfc_approximate_working_set_pages_bucket` (pages being 8kb units is 😬 )

No strong preference btw, any name will do.

Reply:
Changed the name to `lfc_approximate_working_set_size_windows`, and changed the internal metrics label from `duration_minutes` to `duration_seconds` to allow for potential future changes